internal/controller/server/execution/health/execution_test.go

```diff
@@ -120,219 +103,3 @@ func TestUpdateExecutorStatus(t *testing.T) {
 	assert.Equal(t, res.Recieved, true)
 	assert.Nil(t, err)
 }
-
-func TestExecuteJob(t *testing.T) {
-	jobRepoMock := new(job.JobMock)
```

Comment:
These tests are removed from here but not placed somewhere else. Are we losing coverage because of this?

Reply:
It is there in the job package as we removed job-related functions into the job package. you can find it in `internal/controller/server/execution/job/execution_test.go` with all the cases covered